README.md

```diff
@@ -1,10 +1,7 @@
-# <%= url %>
-
-> <%= description %>
+# WordPress project
 
 ## Installation
 
-
 Cloner le dépôt :
```

Comment:
**improvement** add the new way to kickstart the project `composer create-project ...`

Reply:
The issue is that this README.md file will be the one used for the project, it should not contain instructions on how to create a project. Or maybe they could be removed on creation to avoid any confusion.

Reply:
Another idea would be to migrate the new structure to the studiometa/wordpress-project repository and to keep this repository as documentation.

Reply:
I like the idea of replacing the `README` after install. Kind of what we do with `.env.example` or `wp-config-sample` ?